recipes/external-html.md

```diff
@@ -0,0 +1,55 @@
+## External HTML
+
+Developers sometimes face different scenarios while coding and sometimes outputting an index file is unecessary or redudant.
+
+This scenario is usual while developing SPA, where you need a root element on your page to start rendering your application and you have different other custom meta tags.
+
+### Meat and Potatoes
+
+To get started, your `webpack` configuration should be setup and building successfully. Next, have an index file somewhere else, not inside the output folder.
+
+Given the following folder structure:
+
+`/app/index.html`
+`/app/js/app.js`
+`/app/dist` -> being the output folder.
+
+In your `html`
+```html
+<!doctype>
+<html>
+  <body>
+    <main></main>
+    <script src="/assets/bundle.js"></script>
+  </body>
+</html>
+```
+
+You `webpack` config should be:
+
+```js
+const {resolve} = require('fs');
```

Comment:
I guess it should be 'path' instead of 'fs'

Reply:
Typo!! Good catch.